### PR TITLE
Update gin

### PR DESCRIPTION
**Pull request by Tibo-lg**

### CODE REVIEW

**Review by prunelsigma — APPROVED** [review]
image
Ci seems ok so LGTM

**Review by prunelsigma — CHANGES_REQUESTED** [review]
sorry waiting for ci (other repository)

**Review by prunelsigma — APPROVED** [review]
LGTM

**Review by prunelsigma — CHANGES_REQUESTED** [review]
Dockerfile

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ARG ARG_CFD_GO_VERSION=0.2.3
 ARG CFD_GO_ZIP=cfdgo-v${ARG_CFD_GO_VERSION}-alpine_x86_64.zip
-FROM golang:1.14-alpine3.12 as dev
+FROM golang:1.16-alpine3.12 as dev
```
Comment (prunelsigma) [inline Dockerfile:3]: you can remove `3.12` suffix.

Comment (Tibo-lg) [reply Dockerfile:3]: done

**Review by prunelsigma — APPROVED** [review]
LGTM

**Pull request merged**